### PR TITLE
Add mysql executor

### PR DESCRIPTION
**Pull request by gkazakov111**

### CODE REVIEW

File: src/executors/default-with-mysql.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    default: "1"
+  mysql-image-version:
+    type: string
+    default: "5.6"
+docker:
+  - image: golang:<<parameters.golang-image-version>>
+    environment:
+      DB_TEST_URL: root:password@/pac
+  - image: mysql:<<parameters.mysql-image-version>>
+    # db.t2.small allows 45 connections
```
Comment (karsov) [inline src/executors/default-with-mysql.yml:13]: I think we don't need this line.

Comment (gkazakov111) [reply src/executors/default-with-mysql.yml]: Done

File: src/executors/default-with-mysql.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+parameters:
```
Comment (epavlova) [inline src/executors/default-with-mysql.yml:1]: I suggest to add three more parameters to this executor for the environment variables DB_TEST_URL, MYSQL_ROOT_PASSWORD, MYSQL_DATABASE and we can add their current values (respectively root:password@/pac, password, pac) as default values of those parameters.

Comment (gkazakov111) [reply src/executors/default-with-mysql.yml:1]: Done

**Pull request merged**